me.hacktivis.badwolf.yml

```diff
@@ -0,0 +1,28 @@
+app-id: me.hacktivis.badwolf
+runtime: org.gnome.Platform
+runtime-version: "43"
+sdk: org.gnome.Sdk
+command: badwolf
+finish-args:
+ - --socket=wayland
+ - --socket=fallback-x11
+ - --share=network
+ - --device=dri
+ - --share=ipc
+modules:
+ - name: badwolf
+   buildsystem: simple
+   build-commands:
+    - PREFIX=/app ./configure
+    - make install
+    # use standard shortcut name
+    - rename -a badwolf me.hacktivis.badwolf /app/share/icons/hicolor/*/apps/badwolf.*
```

Comment:
we have `rename-icon` in the manifest for that. And this would make the line below unecessary with `rename-destkop-file`

Reply:
i tried that property, it didn't rename the icon

Reply:
maybe because you used it wrong?